views/default.tt

```diff
@@ -262,7 +268,7 @@ start;</pre>
                   <p>
                   It will also be necessary to update your application to talk to your database back-end.  The <code>inventory.pm</code> file is configured to use DBI and $ENV in such a way that it will accept environment variables for your connection information that you pass to it.
                   Once an administrator has created a MySQL database service for you to connect with you can add the following environment variables to your deploymentConfig to ensure all your frontend pods have access to these environment variables.
```

Comment:
"Note: the dancer-mysql.json template creates the DB service and environment variables for you"